scripts/download

```diff
@@ -2,13 +2,3 @@
 set -e
 
 cd $(dirname $0)/..
-
-VERSION=v5.5.1-rancher-1-1
-HASH=dd6dda8081921ac9a91345b1e088787d14a37d0c
-
-if [ "$(sha1sum -b strongswan/strongswan-${VERSION}.tar.gz 2>/dev/null | awk '{print $1}')" != "$HASH" ]; then
-    mkdir -p strongswan
-    echo Downloading strongswan-${VERSION}.tar.gz
-    curl -sLf https://github.com/rancher/strongswan-package/releases/download/${VERSION}/strongswan-${VERSION}.tar.gz > strongswan/strongswan-${VERSION}.tar.gz
-    [ "$(sha1sum -b strongswan/strongswan-${VERSION}.tar.gz | awk '{print $1}')" == "$HASH" ]
```

Comment:
Should we just remove this file altogether?

Reply:
I left the skeleton in to provide a template for future downloads, but yes, we have version control so we might as well delete it.